gluten-celeborn/common/src/main/java/org/apache/spark/shuffle/gluten/celeborn/CelebornShuffleManager.java

```diff
@@ -330,14 +351,18 @@ public <K, C> ShuffleReader<K, C> getReader(
     if (handle instanceof CelebornShuffleHandle) {
       @SuppressWarnings("unchecked")
       CelebornShuffleHandle<K, ?, C> h = (CelebornShuffleHandle<K, ?, C>) handle;
+      CelebornConf readerConf = celebornConf;
+      if (_vanillaCelebornShuffleManager != null) {
+        readerConf = rowBasedCelebornConf;
```

Comment:
When the `_vanillaCelebornShuffleManager` is null if we get row-based, can it happen ?

Reply:
Yes, when gettingReader, ```vanillaCelebornShuffleManager``` is usually null. I am looking to see if there is any way to pass this message.

Reply:
@ulysses-you cc